the-experts-BE/routes/contractor-post.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router({ mergeParams: true });
+const queries = require("../queries/index");
+
+router.post("/", function(req, res, next) {
+  queries.addContractor(
+    req.params.name,
+    req.params.job,
+    (err, response) => {
+      if (err) next(err);
+      res.send(response.rows);
```

Comment:
You should use `res.json()` to make it clear you are sending the data to the client as JSON.